Rubberduck.RibbonDispatcher/Sample Ribbons/SampleRibbon/src/RibbonModel.cls

```diff
@@ -0,0 +1,131 @@
+VERSION 1.0 CLASS
+BEGIN
+  MultiUse = -1  'True
+END
+Attribute VB_Name = "RibbonModel"
+Attribute VB_GlobalNameSpace = False
+Attribute VB_Creatable = False
+Attribute VB_PredeclaredId = False
+Attribute VB_Exposed = False
+Option Explicit
+
+Private Const ModuleName    As String = "RibbonModel."
+Private Const MsgBoxTitle   As String = "Rubberduck Ribbon Demo"
+
+Private Type TRibbonModel
+    BrandingGroup   As RibbonGroup
+    CustomGroup     As RibbonGroup
+    StandardGroup   As RibbonGroup
+End Type
+Private this As TRibbonModel
+
+Private WithEvents mCustomButton1       As RibbonButton
+Attribute mCustomButton1.VB_VarHelpID = -1
+Private WithEvents mCustomButton2       As RibbonButton
+Attribute mCustomButton2.VB_VarHelpID = -1
+Private WithEvents mCustomButton3       As RibbonButton
+Attribute mCustomButton3.VB_VarHelpID = -1
+Private WithEvents mCustomSizeToggle    As RibbonToggleButton
+Attribute mCustomSizeToggle.VB_VarHelpID = -1
+Private WithEvents mCustomOptions       As RibbonDropDown
+Attribute mCustomOptions.VB_VarHelpID = -1
+
+Private WithEvents mBrandingButton      As RibbonButton
+Attribute mBrandingButton.VB_VarHelpID = -1
+Private WithEvents mStandardButton1     As RibbonButton
+Attribute mStandardButton1.VB_VarHelpID = -1
+Private WithEvents mStandardButton2     As RibbonButton
+Attribute mStandardButton2.VB_VarHelpID = -1
+Private WithEvents mStandardOptions     As RibbonDropDown
+Attribute mStandardOptions.VB_VarHelpID = -1
+Private WithEvents mShowAdvancedToggle  As RibbonToggleButton
+Attribute mShowAdvancedToggle.VB_VarHelpID = -1
```

Comment:
Is `varHelpID = -1` of any use?

Reply:
That is robo-code from the VBA environment. I simply perform a Module Export.  I haven't had a chance to actually link in Rubberduck yet, so it's code exporter is likely smarter than my simple one.